server/tilesource/base.py

```diff
@@ -287,58 +548,21 @@ def getRegion(self, width=None, height=None, **kwargs):
         :param height: maximum height in pixels.
```

Comment:
Is `width` and `height` always in maximum resolution pixels (irrespective of the units parameters) ?